src/common/io/socket/client.c

```diff
@@ -25,63 +31,47 @@ static SocketClientStat sckClientStatLocal;
 /***********************************************************************************************************************************
 Object type
 ***********************************************************************************************************************************/
-struct SocketClient
+#define SOCKET_CLIENT_TYPE                                          SocketClient
+#define SOCKET_CLIENT_PREFIX                                        sckClient
+
+typedef struct SocketClient
 {
     MemContext *memContext;                                         // Mem context
     String *host;                                                   // Hostname or IP address
     unsigned int port;                                              // Port to connect to host on
+    String *name;                                                   // Socket name (host:port)
     TimeMSec timeout;                                               // Timeout for any i/o operation (connect, read, etc.)
-};
-
-OBJECT_DEFINE_MOVE(SOCKET_CLIENT);
+} SocketClient;
 
-OBJECT_DEFINE_GET(Host, const, SOCKET_CLIENT, const String *, host);
-OBJECT_DEFINE_GET(Port, const, SOCKET_CLIENT, unsigned int, port);
```

Comment:
They won't be accessible anymore since sckClientNew() returns IoSession and it doesn't make sense for IoSession to expose something so socket-specific. They were just used for logging and were replaced with ioClientName().